include/deal.II/base/polynomial.h

```diff
@@ -585,6 +586,119 @@ namespace Polynomials
     static std::vector<Polynomial<double> >
     generate_complete_basis (const unsigned int p);
   };
+
+
+
+  /**
+   * Polynomials for a variant of Hermite polynomials with better condition
+   * number in the interpolation than the basis from
+   * HermiteInterpolation. This class is only implemented for degree at least
+   * three, $n\geq 3$.
+   *
+   * In analogy to the actual Hermite polynomials this basis evaluates the
+   * first polynomial $p_0$ to 1 in $x=0$ and has a both a zero value and zero
+   * derivative in $x=1$. Likewise, the last polynomial $p_n$ evaluates to 1
+   * in $x=1$ but has zero value and zero derivative in $x=0$. The second
+   * polynomial $p_1$ and the second to last polynomial $p_{n-1}$ represent
+   * the derivative degree of freedom in $x=0$ and $x=1$, respectively. As
+   * such, they are zero in both the end points $x=0, x=1$ and have zero
+   * derivative in the opposite ends, $p_1'(1)=0$ and $p_{n-1}'(0)=0$. As
+   * opposed to the original Hermite polynomials, $p_0$ does not have zero
+   * derivative in $x=0$. The additional degree of freedom is used to make
+   * $p_0$ and $p_1$ orthogonal, which results in a root at $2/7$ for
+   * $n=3$. Furthermore, the extension of these polynomials to higher degrees
```

Comment:
Do you mean "at $x=2/7$"? I presume, this is for only one of the two polynomials, right?